README.md

```diff
@@ -13,41 +13,54 @@ GLUON_SITE_FEEDS='community'
 
 PACKAGES_COMMUNITY_REPO=https://github.com/freifunk-gluon/community-packages.git
 PACKAGES_COMMUNITY_COMMIT=<COMMIT-HASH>
+PACKAGES_COMMUNITY_BRANCH=master
 ```
 
-## Commit guidelines
+If you leave out the `_BRANCH`, the repository default branch is used - your commit has to be on the specified branch.
+
+You should always test and check changes before deploying a new commit, as many people of the community have commit rights and there is currently no test or release process defined.
+
+## Contribution guidelines
+
+### Commit guidelines
 
 * The master branch is protected against the use of `git push --force`.
-* Use Pull Requests if you are uncertain about your patches.
+* Use Pull Requests if you are uncertain about your patches - this is generally a good practice to get feedback from others.
```

Comment:
If someone is unsure, maybe we can introduce adding an `[WIP]` or `[RFC]` tag in front of the PR name?

Reply:
imho using draft PRs is enough.
I also had the experience of forgetting to change the PR name leading to some `WIP: bla` being merged on github in some project - but if you insist I can add something to reflect this?

Reply:
All good, just an idea.

Reply:
Thanks